### PR TITLE
Remove duplicate shortcut definitions

### PR DESCRIPTION
**Pull request by KoBeWi**
Spiritual follow-up to #58847

### CODE REVIEW

File: editor/animation_track_editor.cpp [hunk 1]
```diff
@@ -7312,17 +7312,19 @@ AnimationTrackEditor::AnimationTrackEditor() {
...
-	auto_fit->set_shortcut(ED_SHORTCUT("animation_editor/auto_fit", TTR("Fit to panel"), KeyModifierMask::ALT | Key::F));
+	auto_fit->set_shortcut(ED_GET_SHORTCUT("animation_editor/auto_fit"));
 	bottom_hb->add_child(auto_fit);
 
 	auto_fit_bezier = memnew(Button);
 	auto_fit_bezier->set_flat(true);
 	auto_fit_bezier->set_visible(false);
 	auto_fit_bezier->connect(SceneStringName(pressed), callable_mp(this, &AnimationTrackEditor::_auto_fit_bezier));
-	auto_fit_bezier->set_shortcut(ED_SHORTCUT("animation_editor/auto_fit", TTR("Fit to panel"), KeyModifierMask::ALT | Key::F));
+	auto_fit->set_shortcut(ED_GET_SHORTCUT("animation_editor/auto_fit"));
```
Comment (akien-mga) [inline editor/animation_track_editor.cpp:7327]: do people really copy whole short assignments multiple times and unironically think this is alright :thinking:

**Review by fire — APPROVED** [review]
🫶

**Review by akien-mga — APPROVED** [review]
Looks good to me.

~I'll merge after beta 1 just in case some difference in order of initialization introduce unexpected breakage.~

*Edit:* Actually YOLO, beta 1 will likely only be on Monday so we have time to test such a (seemingly) benign change.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**